src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const obj = Object.entries(robot);
+
+  for (let i = 0; i < obj.length; i++) {
```

Comment:
Why don't you just iterate the object keys here?

Reply:
Yes, Object.keys makes more sense, correcting. I didn't use a forEach because a return in the middle way would be beneficial